### ISSUE TITLE
Freezing After Injection

### ISSUE THREAD

**Issue by tim071** [eliboa/TegraRcmGUI#31]
My device shows up as RCM ok but when trying to inject any payload it hangs at the "Invoking" stage.

**Comment by eliboa** [comment]
Make sure your USB-C cable allows data transfer.

**Comment by tim071** [comment]
Yes even went and bought another cable and still get the same issue. Happens with both portable and Windows installer versions.
